website/src/components/ContentMenu/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Menu from '@icgc-argo/uikit/ContentMenu';
+
+const useMenuHighlight = (schemaRefs, scrollYOffset) => {
+  const [activeItemName, setActiveItemName] = React.useState('');
+
+  React.useEffect(() => {
+    const findActiveSection = () => {
+      const focusedDomEls =
+        schemaRefs.length > 0 &&
+        schemaRefs.filter(ref => {
```

Comment:
`schemaRefs.find` would be much simpler, it'll just give you `undefined` if nothing satisfies your function. Shouldn't even need to check `length`